diff --git a/dist/module.mjs b/dist/module.mjs
index fa00d583d16e6b456dd06920acfb992a4cdbf034..72a303355251ac522fdc44cbbf77f3903987e386 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -116,9 +116,9 @@ Missing \`${theme$1.env}\` env variable, please add it to your \`.env\`.
       }
       return;
     }
-    nuxt.hook("build:before", async () => {
-      await validateLicense({ key, theme: theme$1, dir: nuxt.options.rootDir });
-    });
+    // nuxt.hook("build:before", async () => {
+    //   await validateLicense({ key, theme: theme$1, dir: nuxt.options.rootDir });
+    // });
   }
 });
 
diff --git a/dist/unplugin.mjs b/dist/unplugin.mjs
index e269a72f30f5c9d52f2ef7fce899c260f1520a5a..dfd6c8607dc5add16dd4ce74ac175a3fc7e36663 100644
--- a/dist/unplugin.mjs
+++ b/dist/unplugin.mjs
@@ -156,7 +156,7 @@ const NuxtUIProPlugin = createUnplugin((_options = {}) => {
     ui(options),
     TemplatePlugin(options),
     AppConfigPlugin(appConfig),
-    LicensePlugin(options.license)
+    // LicensePlugin(options.license)
   ].flat(1);
 });
 
